source/docs/software/commandbased/command-based-changes.rst

```diff
@@ -6,7 +6,7 @@ This article provides a summary of changes from the original command-based frame
 Package Location
 ----------------
 
-The new command-based framework is located in the ``wpilibj2`` package for Java, and in the ``frc2`` namespace for C++.  The old command-based framework is still available in the original location. The new framework must be installed using the instructions: :ref:`docs/software/vscode-overview/3rd-party-libraries:WPILib Command Libraries`.
+The new command-based framework is located in the ``wpilibj2`` package for Java, and in the ``frc2`` namespace for C++. The new framework must be installed using the instructions: :ref:`docs/software/vscode-overview/3rd-party-libraries:WPILib Command Libraries`.
```

Comment:
Are we getting rid of the "new" here?

Reply:
given that this article is a comparison of old and new, I don't think we can.

Reply:
Ah yeah, that's fine.